### ISSUE TITLE
Hacks Update

### ISSUE THREAD

**Issue by FelixN0324** [PatheticMustan/ProdigyMathGameHacking#128]
Phaser.GAMES[0].state.states.Boot._metricsManager.gameCompleteDataFactory.gameEventDataBuilder.loggedInPlayer._player.modifiers.maxHearts=1000; ----------> temp2.player.modifiers.maxHearts=1000;

**Comment by Robo-Ren** [comment]
Felix are u a hacker in prodigy?

**Comment by Robo-Ren** [comment]
If you are a hacker then I may need your help

**Comment by Robo-Ren** [comment]
Please help me

**Comment by FelixN0324** [comment]
No, I'm a beginner in hacking. Just fidgeting around with the commands.

**Comment by Robo-Ren** [comment]
Oh do u know someone that could hack in prodigy?

**Comment by FelixN0324** [comment]
Me.
I can hack some of them.

Like:
temp2.player.modifiers.maxHearts=1000 ---> Gives you extra hearts
temp2.player.modifiers.damage=10000 ------> Gives you extra damage on attacks
temp2.player.grade= 1             -------> Gives you grade 1 questions
Besides that nope. I'll try to unpatch more hacks.

**Comment by Robo-Ren** [comment]
Could u gives me extra hearts and give me extra damage?

**Comment by Robo-Ren** [comment]
O I have to give u my user and password?

**Comment by FelixN0324** [comment]
Yes

**Comment by FelixN0324** [comment]
Don't worry, I won't do any harm to your account.

**Comment by Robo-Ren** [comment]
But do u know a hack for my pets could be in level 100?

**Comment by FelixN0324** [comment]
yes

**Comment by FelixN0324** [comment]
for (let PetLoopTest = 0; PetLoopTest < temp2.player.kennel.data.length; PetLoopTest++) { temp2.player.kennel.data[PetLoopTest].level=100 };

**Comment by Robo-Ren** [comment]
Please don't harm my account😭

**Comment by FelixN0324** [comment]
ok, Why would I

**Comment by FelixN0324** [comment]
?

**Comment by Robo-Ren** [comment]
I'm using my cousin acc so he use to hack but now he forgot so he game me his account

**Comment by FelixN0324** [comment]
ok

**Comment by Robo-Ren** [comment]
Could I get all pets?

**Comment by FelixN0324** [comment]
Wait, Let me check

**Comment by Robo-Ren** [comment]
If  u know the hack? Or the trial master gear

**Comment by FelixN0324** [comment]
yes

**Comment by FelixN0324** [comment]
temp2.debugMisc.getAllPets()

**Comment by FelixN0324** [comment]
??? I don't know the trial master hack.

**Comment by Robo-Ren** [comment]
Dude u really pro wow🎂

**Comment by Robo-Ren** [comment]
Oh

**Comment by Robo-Ren** [comment]
ok last question

**Comment by FelixN0324** [comment]
ok, what is it?

**Comment by Robo-Ren** [comment]
Do u know the hack for all every clothing

**Comment by Robo-Ren** [comment]
?

**Comment by FelixN0324** [comment]
Yes

**Comment by Robo-Ren** [comment]
Ok

**Comment by FelixN0324** [comment]
It's long, so be patient

**Comment by Robo-Ren** [comment]
Ok

**Comment by Robo-Ren** [comment]
Are u going to give me max coins?

**Comment by Robo-Ren** [comment]
Ok my account is

**Comment by Robo-Ren** [comment]
Chasethebag11

**Comment by Robo-Ren** [comment]
and my password is Hello

**Comment by Robo-Ren** [comment]
please don't do anything

**Comment by FelixN0324** [comment]
(() => {const bootData = temp1.object.n.c[0].exports.a.instance.game.state.states.Boot._gameData;
    const backpackData = temp1.object.n.c[0].exports.a.instance.prodigy.player.backpack.data;
    const playerObject = temp1.object.n.c[0].exports.a.instance.prodigy.player;
let a = ["outfit", "hat", "boots", "weapon", "spellRelic", "fossil", "follow"];
    for (let u of a) {
        backpackData[u] = [];
        for (let i in bootData[u]) {
            backpackData[u][i] = {"ID": bootData[u][i].ID};
        }
    }
    backpackData.currency = [];
    for (let i in bootData.currency) {
        backpackData.currency[i] = {"ID": bootData.currency[i].ID, "N": 99999999};
    }
    backpackData.item=[];
    for (let i in bootData.item) {
        backpackData.item[i] = {"ID": bootData.item[i].ID, "N": 99999999};
    }
    playerObject.house.data.items = []
    for (i in bootData.dorm) {
        playerObject.house.data.items[bootData.dorm[i]["ID"]] = {A: [], N: 999}
    }
})()

**Comment by FelixN0324** [comment]
ok

**Comment by FelixN0324** [comment]
No harm will be done

**Comment by Robo-Ren** [comment]
Oki

**Comment by Robo-Ren** [comment]
Could we be friends

**Comment by Robo-Ren** [comment]
When could I log in to my account to check?

**Comment by FelixN0324** [comment]
Yeah

**Comment by Robo-Ren** [comment]
Oki

**Comment by FelixN0324** [comment]
Wait

**Comment by Robo-Ren** [comment]
Ok

**Comment by FelixN0324** [comment]
done! You're now level 100 and so are your pets!


But, the others are not permanent so do you need more instructions to do it.

**Comment by Robo-Ren** [comment]
Thank u

**Comment by Robo-Ren** [comment]
which instructions?

**Comment by FelixN0324** [comment]
do you want the instructions to the other hacks? They are not permanent so you have to do it.

**Comment by Robo-Ren** [comment]
I'm in lvl 32 still

**Comment by Robo-Ren** [comment]
55*

**Comment by Robo-Ren** [comment]
ok

**Comment by Robo-Ren** [comment]
Why I'm not in lvl 100😭

**Comment by Robo-Ren** [comment]
Dude I trust u why I'm not in lvl 100 and my pets

**Comment by FelixN0324** [comment]
????

**Comment by FelixN0324** [comment]
It should work.

**Comment by Robo-Ren** [comment]
It did not work

**Comment by FelixN0324** [comment]
Ok, let me try again.

**Comment by FelixN0324** [comment]
You'll get a Crystal Monster though

**Comment by Robo-Ren** [comment]
No I do not have crystal monster

**Comment by Robo-Ren** [comment]
Ok

**Comment by FelixN0324** [comment]
It's in the mess

**Comment by Robo-Ren** [comment]
What is that

**Comment by FelixN0324** [comment]
You will have to delete some of the mess to see it
the pet kennel has a limited amount of pets

**Comment by FelixN0324** [comment]
To see the Crystal Monster

**Comment by Robo-Ren** [comment]
So do I need to delete my pets?

**Comment by FelixN0324** [comment]
It's the one that is always loading because Prodigy doesn't have an face for it

**Comment by FelixN0324** [comment]
Some of it on page 8 only

**Comment by Robo-Ren** [comment]
Ok

**Comment by Robo-Ren** [comment]
I'm going to check

**Comment by Robo-Ren** [comment]
No wayyy

**Comment by Robo-Ren** [comment]
Thank u

**Comment by Robo-Ren** [comment]
Do u know the buddies?

**Comment by FelixN0324** [comment]
Until you see the Crystal Monster.

**Comment by FelixN0324** [comment]
Yes

**Comment by Robo-Ren** [comment]
Yes I see it

**Comment by Robo-Ren** [comment]
could u do the buddies please

**Comment by Robo-Ren** [comment]
THANK UUU

**Comment by Robo-Ren** [comment]
Dude u are really a hacker thank u

**Comment by Robo-Ren** [comment]
Could u do it to my other acc?

**Comment by FelixN0324** [comment]
YES!

**Comment by Robo-Ren** [comment]
I will defenelity appreciate it

**Comment by Robo-Ren** [comment]
thank u

**Comment by Robo-Ren** [comment]
but please not this acc dont try to harm it

**Comment by FelixN0324** [comment]
You're first account is becoming level 100

**Comment by Robo-Ren** [comment]
because the one that I'm about to give u is my cousin

**Comment by Robo-Ren** [comment]
for real wooooowww

**Comment by Robo-Ren** [comment]
thank u
I'm going to eat

**Comment by Robo-Ren** [comment]
when I finish

**Comment by Robo-Ren** [comment]
I give u my cousin account for u could level up the pets and the

**Comment by FelixN0324** [comment]
Oh You also got a ton of conjure cubes! (Something I've never seen happen in a battle.

**Comment by Avn1114** [comment]
Agh I'm getting so many EMAILS HELP ME

**Comment by FelixN0324** [comment]
9 conjure cubes only.

**Comment by Robo-Ren** [comment]
Oki

**Comment by Robo-Ren** [comment]
Do i give u my cousin thing I mean my account

**Comment by FelixN0324** [comment]
I got you all the buddies

**Comment by FelixN0324** [comment]
Yes the one with ROBO-VAN

**Comment by FelixN0324** [comment]
Also check your items! You can throw as much water balloons and snowballs along with MAX morph marbles to keep you for years!

**Comment by Robo-Ren** [comment]
Thank u

**Comment by FelixN0324** [comment]
8 conjure cubes. I can show you how to hack the damage and health If you want.

**Comment by Robo-Ren** [comment]
Are u going to be active tomorrow?

**Comment by Robo-Ren** [comment]
maybe tomorrow

**Comment by Robo-Ren** [comment]
But sure

**Comment by Robo-Ren** [comment]
yes

**Comment by FelixN0324** [comment]
Maybe. I'll have online classes. But

**Comment by FelixN0324** [comment]
I'll try to be there. Do you want to play with a friend of mine? (Maybe though)

**Comment by Robo-Ren** [comment]
Yea I'm doing online classes

**Comment by Robo-Ren** [comment]
okay

**Comment by Robo-Ren** [comment]
Which friend

**Comment by FelixN0324** [comment]
Where do you go to school?

**Comment by FelixN0324** [comment]
A friend of mine.

**Comment by Robo-Ren** [comment]
Elizabeth

**Comment by Robo-Ren** [comment]
school 25🤦‍♂️

**Comment by FelixN0324** [comment]
In guilford County?

**Comment by Robo-Ren** [comment]
No😂

**Comment by FelixN0324** [comment]
Then which county?

**Comment by Robo-Ren** [comment]
Usa

**Comment by Robo-Ren** [comment]
united states

**Comment by FelixN0324** [comment]
No not country, County

**Comment by Robo-Ren** [comment]
Emm elizabeth?

**Comment by FelixN0324** [comment]
Or state?

**Comment by Robo-Ren** [comment]
new jersey

**Comment by FelixN0324** [comment]
ohhhhhhhhhhhhh.

**Comment by FelixN0324** [comment]
That makes sense.

**Comment by Robo-Ren** [comment]
Dude thank uuu

**Comment by FelixN0324** [comment]
Ok, see you later or tomorrow.

**Comment by Robo-Ren** [comment]
I log in and saw it omggg

**Comment by Robo-Ren** [comment]
bye at what time we going to play tomorrow?

**Comment by FelixN0324** [comment]
You're welcome. All buddies and pets and level 100?

**Comment by Robo-Ren** [comment]
Thank u I got the swift hood lets goo

**Comment by FelixN0324** [comment]
Around 8 am-4 pm

**Comment by Robo-Ren** [comment]
could u do my other acc tomorrow?

**Comment by FelixN0324** [comment]
sure

**Comment by Robo-Ren** [comment]
oki

**Comment by Robo-Ren** [comment]
Ok

**Comment by FelixN0324** [comment]
We'll meet back here.

**Comment by Robo-Ren** [comment]
goodbye see u tomorrow

**Comment by Robo-Ren** [comment]
Ok

**Comment by FelixN0324** [comment]
See you tommorow

**Comment by FelixN0324** [comment]
Until then

**Comment by Robo-Ren** [comment]
Ok bye

**Comment by FelixN0324** [comment]
This message is for tomorrow.

Meet me in the world "Amphibia" on Prodigy. I'll be the earliest is at 8 am.

**Comment by Robo-Ren** [comment]
Ok

**Comment by Diveodile** [comment]
I posted the video on how to hack Prodigy:
https://www.youtube.com/watch?v=1dM8m-gs7hI

**Comment by Robo-Ren** [comment]
Div but it won't let me inspect

**Comment by Robo-Ren** [comment]
do we have to use a Google account

**Comment by TNThacker2015** [comment]
Don't use temp2, use temp1.object.n.c[0].exports.a.prodigy

**Comment by Magmischief** [comment]
@FelixN0324 hack my acc plz

**Comment by Magmischief** [comment]
i'm very desperate

**Comment by Robo-Ren** [comment]
Why?

**Comment by Magmischief** [comment]
i need membership and all gear

**Comment by Magmischief** [comment]
NOW

**Comment by Robo-ren** [comment]
Well u will need to wait to tomorrow

**Comment by Robo-ren** [comment]
Morning at 8:am

**Comment by Magmischief** [comment]
Bruh

**Comment by Magmischief** [comment]
I CAN'T WAIT

**Comment by Robo-ren** [comment]
Yep he is gonna hack my different account

**Comment by Robo-ren** [comment]
U will have to wait alot

**Comment by Robo-ren** [comment]
Wanna play prodigy?

**Comment by FelixN0324** [comment]
Hi, Are you still there?
I'll be here for a limited amount of time today.

**Comment by Diveodile** [comment]
> Bruh

I do not want to alarm you, but there already is a Magmischief in Github!

**Comment by TNThacker2015** [comment]
Github is not a place to talk casually. I'll close this for the time being.

**Comment by Robo-ren** [comment]
Hello

**Comment by PatheticMustan** [comment]
As @TNThacker2015 said, Github is NOT a place for you to chat. Unless you can actually contribute information relevant to the present issue, please refrain from impulsively posting whatever comes to your mind.
